filer/templates/admin/filer/breadcrumbs.html

```diff
@@ -11,7 +11,7 @@
             {% if ancestor_folder.label %}
                 {{ ancestor_folder.label }}
             {% else %}
-                {% if ancestor_folder.name != "unfiled files" %}
+                {% if ancestor_folder.name != _("Unsorted Uploads") %}
```

Comment:
So why don't we just make ``ancestor_folder.label`` or ``ancestor_folder.name`` return the correct translated string? Then all this logic would not be necessary.